velox/substrait/tests/JsonToProtoConverter.h

```diff
@@ -24,4 +24,7 @@ class JsonToProtoConverter {
   static void readFromFile(
       const std::string& msgPath,
       google::protobuf::Message& msg);
+
+  /// Convert Protobuf message to Json.
+  static std::string messageToJson(const google::protobuf::Message& message);
```

Comment:
This method doesn't appear to be used in this PR. Do you need this for debugging? If so, let's move this into a separate PR and add a test.